packages/features/bookings/lib/handleNewBooking/createBooking.ts

```diff
@@ -182,6 +182,22 @@ async function saveBooking(
       await tx.app_RoutingForms_FormResponse.update(reroutingFormResponseUpdateData);
     }
 
+    const organizerCredentials = await tx.credential.findMany({
+      where: { userId: organizerUser.id },
+      select: { id: true },
+    });
+
+    if (organizerCredentials.length > 0) {
+      await tx.calendarCache.updateMany({
+        where: {
+          credentialId: {
+            in: organizerCredentials.map((cred) => cred.id),
+          },
+        },
+        data: { stale: true },
+      });
+    }
+
```

Comment:
DevinAi, extract this to a function a repository that receives a prisma client or transacion as parameter. Also are we sure we only need the organizer here? What about round robin events or collective events?